faction/pom.xml

```diff
@@ -2,26 +2,22 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.feildmaster.module.faction</groupId>
     <artifactId>FactionChannel</artifactId>
-    <version>1.2</version>
+    <version>2.0</version>
     <name>FactionChannel</name>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
     <repositories>
         <repository>
-            <id>bukkit-repo</id>
-            <url>http://repo.bukkit.org/content/groups/public/</url>
-        </repository>
-        <repository>
-            <id>fm-repo</id>
-            <url>http://repo.feildmaster.com/</url>
```

Comment:
Your repo resulted in a 404 and fieldmaster.com has been replaced by a page to create ball fields.